h/accounts/test/views_test.py

```diff
@@ -830,30 +829,28 @@ def test_activate_event_when_validation_succeeds(event,
 
 
 @pytest.mark.usefixtures('subscriptions_model')
-def test_profile_looks_up_by_logged_in_user(authn_policy, user_model):
-    """
-    When fetching the profile, look up email for the logged in user.
+def test_profile_looks_up_by_logged_in_user():
```

Comment:
I'd just remove this test. It's no longer testing a collaboration with another component.